BlazorStaticWebsite/Components/Blog/PostsList.razor

```diff
@@ -1,7 +1,7 @@
 @using System.Globalization
 @using BlazorStatic
 @using BlazorStatic.Services
-@inject BlogService<FrontMatter> BlogService
+@inject BlazorStaticContentService<BlogFrontMatter> BlazorStaticContentService
```

Comment:
i think `BlazorStaticContentService` (the variable name) should be changed to camel case to avoid conflicts with the service class

Reply:
Done everywhere